packages/realm/binding/platform.hpp

```diff
@@ -19,6 +19,7 @@
 #pragma once
 
 #include <string>
+#include "realm/object-store/shared_realm.hpp"
```

Comment:
As this is referring to an include outside of the JS SDK code-base:

```suggestion
#include <realm/object-store/shared_realm.hpp>
```